commercetools-models/src/test/java/io/sphere/sdk/extensions/commands/ExtensionUpdateCommandIntegrationTest.java

```diff
@@ -0,0 +1,95 @@
+import io.sphere.sdk.test.IntegrationTest;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.function.UnaryOperator;
+
+import static io.sphere.sdk.subscriptions.SubscriptionFixtures.assumeHasAzureFunctionUrl;
+import static io.sphere.sdk.subscriptions.SubscriptionFixtures.azureFunctionUrl;
+import static io.sphere.sdk.test.SphereTestUtils.asList;
+import static io.sphere.sdk.test.SphereTestUtils.randomKey;
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+public class ExtensionUpdateCommandIntegrationTest extends IntegrationTest {
+
+
+    private static String AZURE_FUNCTION_URL;
+
+    @BeforeClass
+    public static void deleteAllExtensions() {
+        assumeHasAzureFunctionUrl();
+        AZURE_FUNCTION_URL = azureFunctionUrl();
+        PagedQueryResult<Extension> results = client().executeBlocking(ExtensionQuery.of());
+        results.getResults()
+                .stream()
+                .map(ExtensionDeleteCommand::of)
+                .forEach(client()::executeBlocking);
+    }
+
+    @Test
+    public void setKey() {
+
+        withExtension(client(), extension -> {
+            final String newKey = randomKey();
+            Extension updatedExtension = client().executeBlocking(ExtensionUpdateCommand.of(extension, SetKey.of(newKey)));
+            assertThat(updatedExtension.getKey()).isEqualTo(newKey);
+            return updatedExtension;
+        });
+    }
+
+
+    @Test
+    public void changeTriggers(){
+
+        withExtension(client(), extension -> {
+            final List<Trigger> triggers = asList(TriggerBuilder.of(Cart.referenceTypeId(), asList(TriggerType.CREATE)).build());
+            Extension updatedExtension = client().executeBlocking(ExtensionUpdateCommand.of(extension, ChangeTriggers.of(triggers)));
+            assertThat(updatedExtension.getTriggers()).containsAll(triggers);
+            return updatedExtension;
+        });
+
+    }
+
+
+    @Test
+    public void changeDestination(){
+
+        withExtension(client(), extension -> {
+            Destination destination = HttpDestinationBuilder.of(AZURE_FUNCTION_URL, AzureFunctionsAuthenticationBuilder.of(randomKey()).build()).build();
+            Extension updatedExtension = client().executeBlocking(ExtensionUpdateCommand.of(extension, ChangeDestination.of(destination)));
+            assertThat(updatedExtension.getDestination()).isEqualTo(destination);
+            return updatedExtension;
+        });
+
+    }
+
+
+    public void withExtension(final BlockingSphereClient client, final ExtensionDraft extensionDraft, UnaryOperator<Extension> mapper) {
+
+        Extension extension = client.executeBlocking(ExtensionCreateCommand.of(extensionDraft));
+        Extension result = mapper.apply(extension);
+        client.executeBlocking(ExtensionDeleteCommand.of(result));
+    }
+
+    public void withExtension(final BlockingSphereClient client, UnaryOperator<Extension> mapper) {
```

Comment:
`final` is missing for the last parameter.

Reply:
👍 good catch